Makefile

```diff
@@ -423,13 +456,19 @@ verify-docs: site-build clean-html
 verify-codespell codespell:
 	$(Q) codespell
 
+verify-licenses: licenses
+	$(Q)./scripts/build/verify-licenses.sh -v $(LICENSE_PATH)
+
 #
 # targets for installing dependencies
 #
 
 install-ginkgo:
 	$(Q)$(GO_INSTALL) -mod=mod github.com/onsi/ginkgo/v2/ginkgo
 
+install-go-licenses:
+	$(Q)$(GO_INSTALL) github.com/google/go-licenses@latest
```

Comment:
suggestion: pin to a specific version. Prevents possible CI failures just because something changed in go-licenses